src/model/race.js

```diff
@@ -0,0 +1,44 @@
+//SM: Seriously?  Enums?  What is the right model to do this in JS?  Is the right answer just strings everwhere?
```

Comment:
We can do it in Typescript and use enums and it would clean it all up quite a bit.
https://www.typescriptlang.org/docs/handbook/enums.html

I started to write some interfaces, but I was also lazy about it and didn't know how things _actually_ worked in 13th age. https://github.com/ctmoses/projectflimflam/blob/master/src/types.ts